packages/verified-fetch/src/utils/get-peer-id-from-string.ts

```diff
@@ -0,0 +1,12 @@
+import { peerIdFromCID, peerIdFromString } from '@libp2p/peer-id'
+import { CID } from 'multiformats/cid'
+import type { PeerId } from '@libp2p/interface'
+
+export function getPeerIdFromString (peerIdString: string): PeerId {
```

Comment:
we had three places where we were doing libp2p peerId string parsing, so I pulled this into a single place.